linux/hci/hci.go

```diff
@@ -411,6 +423,24 @@ func (h *HCI) sktLoop() {
 				_ = logger.Error("skt: %v", err)
 			} else {
 				log.Printf("skt: %v", err)
+				// close current hci socket
+				err = h.Close()
+				if err != nil {
+					return
+				}
```

Comment:
If an error occurs, this socket loop is closed. How should it be retried?

Reply:
@zakkie 
Thanks!
Indeed, if the socket cannot be closed, it seems better to continue the process as the socket is still alive.